crates/wasm-compose/example/service.wit

```diff
@@ -1,4 +1,4 @@
-package example:service
+package example:service@0.1.0
```

Comment:
What does this version mean?

Reply:
Right now `cargo-component` automatically versions the components it builds, even if the `package` isn't explicitly versioned in WIT (if it is, it must match what's in `Cargo.toml`). It does this to prevent two sources of truth for the version of the component you're building.

However, the `wasmtime::component::generate!` macro doesn't support specifying a version independent of the WIT; thus we add it here to ensure that the Wasmtime bindings look for the right export from the component.